### PR TITLE
feat: adds legacy button to school picker CUR-248

### PR DESCRIPTION
**Pull request by k-huggs**
## Description

- adds link to teacher hub page to school picker

## Issue(s)

Fixes #CUR-248

## How to test

1. Go to https://deploy-preview-1888--oak-web-application.netlify.thenational.academy/beta/teachers/curriculum
2. Click on school picker
3. You should see link to legacy downloads page

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:

Screenshot 2023-09-08 at 12 03 35

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner

### CODE REVIEW

File: src/components/SubjectPhasePicker/SubjectPhasePicker.tsx [hunk 1]
```diff
@@ -320,6 +322,20 @@ const SubjectPhasePicker: FC<SubjectPhasePickerData> = ({
                     />
                   </ButtonContainer>
                 ))}
+                <Box $mt={24}>
+                  <OakLink
+                    page={"oak-curriculum"}
+                    $textDecoration={"underline"}
+                    $font={"body-2-bold"}
+                  >
+                    See curricula from previous academic years{" "}
```
Comment (JBR90) [inline src/components/SubjectPhasePicker/SubjectPhasePicker.tsx:331]: remove {" "}

**Review by JBR90 — APPROVED** [review]
Looks good, one small comment

File: src/components/SubjectPhasePicker/SubjectPhasePicker.tsx (hunk 1 above)
Comment (k-huggs) [reply src/components/SubjectPhasePicker/SubjectPhasePicker.tsx]: removed ✅

**Review by rightsaidjames — APPROVED** [review]
Appearance (including padding above and below) is correct across all breakpoints, link takes you to the correct place - LGTM!

**Review by HelenHarlow — APPROVED** [review]
Looks good to me! Just FYI I need to check in with Hayley about the copy on the button, she's off sick today but will do it as soon as she's back

**Pull request merged**